rest-api-spec/src/main/resources/rest-api-spec/test/ingest/70_bulk.yaml

```diff
@@ -76,11 +76,12 @@ setup:
   - do:
       nodes.stats:
         metric: [ ingest ]
-  - gte: {nodes.$master.ingest.total.count: 1}
+  #we can't assert anything here since we might have more than one node in the cluster
```

Comment:
right, I think there is no way of asserting anything useful, unless we can send the bulk requests to specific nodes in the yaml tests.